priv/template/brunch-config.js

```diff
@@ -0,0 +1,31 @@
+exports.config = {
+  // See http://brunch.io/#documentation for docs.
```

Comment:
We should move this line up and add a couple comments about what is brunch and so on since we are making this decision for the user.